### PR TITLE
Render missing web urls

### PR DESCRIPTION
**Pull request by donni106**
feat(InfoCard): merge all available web urls to render

- added attributes to queries to request all necessary data
- created a method to merge all available web urls to render in `InfoCard` for events, points of interest and tours

feat(web urls): filter descriptions starting with "url"

- added a check for descriptions starting with "url" because we do not want to present web urls behind an "url" or "urlInformation" or "urlVideo" or "urlVideoPreview"
  - in that case, we render the just the `url` instead of the `description`

SVA-146 (_please see comment in the ticket for screenshots_)

### CODE REVIEW

File: src/components/screens/InfoCard.js [hunk 1]
```diff
@@ -67,6 +67,24 @@ const contactView = (contact) => (
   </View>
 );
 
+const mergeWebUrls = ({ webUrls, contact, contacts }) => {
+  const mergedWebUrls = [...webUrls];
```
Comment (digorath) [inline src/components/screens/InfoCard.js:71]: is `webUrls` ensured to not be `undefined` here?
trying to spread `undefined` causes an error.

you can avoid it with
```
const mergedWebUrls = [...(webUrls ?? [])];
```
or with
```
const mergedWebUrls = webUrls ? [...webUrls] : [];
```
of which i prefer the second version for readability reasons.

**Review by digorath — CHANGES_REQUESTED** [review]
One small thing to fix.

File: src/components/screens/InfoCard.js (hunk 1 above)
Comment (donni106) [reply src/components/screens/InfoCard.js]: done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/170/commits/d919904fe6442110d647dfd00b57a9d280e445f2

**Review by digorath — APPROVED** [review]

**Review by SoleCincis — APPROVED** [review]

File: src/queries/eventRecords.js [hunk 2]
```diff
@@ -102,6 +102,22 @@ export const GET_EVENT_RECORDS_AND_CATEGORIES = gql`
...
+        fax
+        webUrls {
+          url
+          description
+        }
+      }
+      webUrls: urls {
+        url
+        description
+      }
```
Comment (donni106) [inline src/queries/eventRecords.js:120]: we missed someting:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/195/files

**Pull request merged**